modules/custom/cu_users/cu_users.module

```diff
@@ -303,6 +303,7 @@ function cu_users_user_presave(&$edit, $account, $category) {
 
       // Filter the roles being saved to make sure the user always has one left.
       $remaining_core_roles = array_filter($edit['roles'], function ($val2) use ($core_role_ids) {
+        $val2 = (int) $val2;
```

Comment:
Why is this in here?  This entire function is ridiculously over complicated and has hardcoded roles https://github.com/CuBoulder/express/blob/f3835f792569c27a118d8eb23559a1fa12762690/modules/custom/cu_users/cu_users.module#L282

Reply:
I think this was part of the earlier troubleshooting that I did that somehow ended up being pushed up in the PR. It's definitely not needed here.